Docs/source/usage/parameters.rst

```diff
@@ -1147,7 +1151,8 @@ Numerics and algorithms
     a sine squared and is applied to the fields on the outer half of the guards.
```

Comment:
This input parameter can be removed. There's probably no need to include it in `BackwardCompatibility` since it has only been used in one case so far.

Reply:
great! thanks a lot!